src/pages/index.page.tsx

```diff
@@ -70,7 +70,19 @@ const Home = () => {
             </div>
           </div>
 
-          <div className="card border-primary bg-primary-content border shadow-xl">
+          <form
+            className="card border border-primary bg-primary-content shadow-xl"
+            onSubmit={(e) => {
+              e.preventDefault();
+              if (!inputState.name || !inputState.code)
+                return toast.error("Please fill out all fields", {
+                  id: "missing-fields",
+                });
+              joinRoom.mutate({
+                ...inputState,
+              });
+            }}
+          >
```

Comment:
good catch

Reply:
honestly might even wanna use react hook form here. depends if we actually end up needing the dependency